src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -10716,6 +11190,7 @@ .maxstack  2
     IL_0026:  br.s       IL_002a
     // sequence point: break;
     IL_0028:  br.s       IL_002a
+    // sequence point: <hidden>
```

Comment:
Is there a general rule we should remember for why the hidden sequence point was necessary?

Reply:
It was necessary because it precedes compiler-generated code that doesn't necessarily correspond to the user code of the immediately preceding IL.